src/static/js/cockpit.js

```diff
@@ -33,6 +59,21 @@
     });
   };
 
+  Cockpit.prototype.loadUi = function(done) {
```

Comment:
Can we change loadUi to one of the following: loadTheme, loadUiTheme, loadSkin, loadUiSkin.  Skin and Theme customization terms that people are familiar with.